lib/marked.js

```diff
@@ -1340,7 +1340,20 @@ function merge(obj) {
 }
 
 function splitCells(tableRow, count) {
-  var cells = tableRow.replace(/([^\\])\|/g, '$1 |').split(/ +\| */),
+  var row = tableRow.replace(/\|/g, function (match, offset, str) {
```

Comment:
Can you add a comment above this line/function explaining its purpose? (It looks like the goal is to ensure that every cell-delimiting pipe has a space before it, so that you can split cells on `' |'`).